App.js

```diff
@@ -0,0 +1,83 @@
+const header = document.querySelector(".header");
+// const section = document.querySelector(".Section-1");
+// document.section.append(header);
+
+//hanburger menu in js
```

Comment:
[OPTIONAL] Please avoid leaving unused codes as comments in your code base since it makes it bulky thereby reducing performance.